src/pkg/provisioner/run_script_step.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package provisioner
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+)
+
+type runScriptStep struct {
+	BuildContext string
+	Path         string
+	Env          string
+}
+
+func (s *runScriptStep) run(runState *state) error {
+	log.Printf("Executing script %q...", s.Path)
+	script := filepath.Join(runState.dir, s.BuildContext, s.Path)
+	cmd := exec.Command("/bin/bash", script)
```

Comment:
Maybe we can share some code with the `systemd.go:stop` function if that pattern is similar.

Reply:
Added a utils.RunCommand() function to use in these places.